README.md

```diff
@@ -86,9 +86,11 @@ mvn compile quarkus:dev
 
 ```bash
 cd barista-node
-npm start
+KAFKA_BOOTSTRAP_SERVERS=localhost:9092 npm start
```

Comment:
I understand the request to make it configurable, and include instructions in the doc of how to set it, but for the quarkus barista you don't have to set it in the default case, so I think it would be good if the Node.js had a default as well and you only need to set it if you were not using the localhost.